### ISSUE TITLE
Doesn't work on the editor background anymore for me

### ISSUE THREAD

**Issue by WarriorKat** [RichardLuo0/vscode-frosted-glass-theme#2]
It was working yesterday after clicking update and enable a few times but now the background color and backdrop filter won't apply to the background of the editor, only to a few prompts.

![image](https://user-images.githubusercontent.com/34631181/141774290-f9d76f11-1bbe-434f-8216-29a301760ea1.png)
![image](https://user-images.githubusercontent.com/34631181/141774385-b93d7abd-ca9e-4a5d-94e6-fc6fd7e04620.png)

I have tried 

- Clicking Update Frosted Glass Theme _(tons of times)_
- Disabling then enabling it then restarting vsc with both the button and by closing vsc and opening in administrator mode _(tons of times)_
- Uninstalling and installing again
- Installing few of the older versions of the extension
- Applying different themes, then clicking update, enable and restarting in different order combinations.

I also don't have any other extension that could interfere with vsc's appearance, except for one downloaded and disabled theme "Monokai".
I am using windows and vsc 1.62.2

**Comment by RichardLuo0** [comment]
Could you plz take a screenshot of the elements tab of developer tools? It is under help-> developer tools.

**Comment by WarriorKat** [comment]
![image](https://user-images.githubusercontent.com/34631181/141784964-7c605cf7-f4a3-4e82-8237-4d63b8db08c3.png)

**Comment by RichardLuo0** [comment]
Are you sure you have changed this option and then run "update frosted glass theme"?
![image](https://user-images.githubusercontent.com/22096448/141786207-b14e67f4-7089-4529-bf25-d1dab080ea4d.png)

**Comment by WarriorKat** [comment]
I tried changing that value just now and ran >Update Frosted Glass Theme and then pressed Restart Visual Studio Code, ran update, ran enable, closed it, opened it up, ran enable and ran update once again and its still the same.

The only probable cause I can think of might be my Windows 10 version. The update is stuck since I reinstalled it so it's an ancient build and sometimes things work inproperly because of it.

If you're clueless about it I don't mind closing the issue.
